lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -1136,9 +1158,7 @@ void shouldRemoveOperationalProfileIfMatchingGpg45ProfileIsPresent() throws Exce
     }
 
     @Test
-    void
-            shouldReturnMitigationJourneyStepResponseIfResetIdentityTrueCiMitigationJourneyStepPresentAndMitigationJourneyStepPresent()
```

Comment:
I am more concerned by this test - it looked like it was doing what the original name said because the mitigation journey response set on line 1152 happened to be the reprove response which is also returned by a different code path.
Hopefully the original test name is wrong/out of date